### ISSUE TITLE
"Upcoming Release Changes" PR workflows not running

### ISSUE THREAD

**Issue by YaroShkvorets** [graphprotocol/graph-tooling#1872]
i.e https://github.com/graphprotocol/graph-tooling/pull/1864

As a result final checks before the release are not performed, including CLI RC release.

The issue here is that GitHub doesn't trigger workflows from events created by other workflows to avoid infinite loop: https://github.com/peter-evans/create-pull-request/issues/48

Possible workarounds:
- create this PR with different PAT token
- cron ?
- manual dispatch ?

**Comment by 0237h** [comment]
I believe this is a duplicate of #1757 but hadn't had the chance to look at it more in depth. Do you want to close the other one in favor of this one and tackle it or we keep the original ?

**Comment by YaroShkvorets** [comment]
No, that's different one. The rate limit one was just failing some checks. This one doesn't even start checks, for example: https://github.com/graphprotocol/graph-tooling/pull/1887

**Comment by 0237h** [comment]
Oh right I see, my bad :+1:
